pkgs/desktops/gnome-3/games/aisleriot/default.nix

```diff
@@ -1,23 +1,31 @@
-{ stdenv, fetchurl, pkgconfig, gnome3, intltool, itstool, gtk3
-, wrapGAppsHook, librsvg, libxml2, desktop-file-utils
-, guile_2_0, libcanberra-gtk3 }:
+{ stdenv, fetchFromGitLab, pkgconfig, gnome3, intltool, itstool, gtk3
+, wrapGAppsHook, meson, librsvg, libxml2, desktop-file-utils, pysolfc
+, guile_2_0, libcanberra-gtk3, ninja, qt5Full, pkgs, appstream-glib, yelp-tools }:
 
 stdenv.mkDerivation rec {
   pname = "aisleriot";
-  version = "3.22.9";
+  version = "3.22.11";
 
-  src = fetchurl {
-    url = "mirror://gnome/sources/${pname}/${stdenv.lib.versions.majorMinor version}/${pname}-${version}.tar.xz";
-    sha256 = "0yzdh9cw5cjjgvfh75bihl968czlgfmpmn1z0fdk88sgvpjgzwji";
+  src = fetchFromGitLab {
+    owner = "GNOME";
+    repo = pname;
+    domain = "gitlab.gnome.org";
+    rev    = "${version}";
+    sha256 = "1asm0y6485xqsysdg586y3hzz8bhxqwnc82k6vhfnxpxz7l62qa1";
   };
 
-  configureFlags = [
-    "--with-card-theme-formats=svg"
-    "--with-platform=gtk-only" # until they remove GConf
+  nativeBuildInputs = [
+    wrapGAppsHook
+    meson
+    pkgconfig
+    intltool
```

Comment:
intltool not used anymore: https://gitlab.gnome.org/GNOME/aisleriot/-/issues/13

Reply:
Removed from dependency list. Thnaks